src/components/SwapTokenInput.tsx

```diff
@@ -0,0 +1,300 @@
+            variant="body2"
+            color="text.secondary"
+            textAlign="end"
+          >
+            ≈$
+            {commify(formatBNToString(inputValueUSD, 18, 2))}
+          </Typography>
+        </Box>
+      </Box>
+
+      <StyledPopper
+        open={open}
+        anchorEl={anchorEl}
+        placement="bottom-start"
+        sx={{ width: popOverWidth ? popOverWidth : 400 }}
+      >
+        <ClickAwayListener onClickAway={handleClose}>
+          <Box height="100%" borderRadius="6px">
+            <Autocomplete
+              open={open}
+              options={tokens}
+              popupIcon={null}
+              renderInput={(params) => (
+                <TextField
+                  {...params}
+                  variant="standard"
+                  data-testid="searchTermInput"
+                  ref={params.InputProps.ref}
+                  InputProps={{
+                    startAdornment: <Search />,
+                  }}
+                  inputProps={params.inputProps}
+                  placeholder="Search name"
+                  autoFocus
+                />
+              )}
+              onChange={(event, newValue, reason) => {
+                if (
+                  event.type === "keydown" &&
+                  (event as React.KeyboardEvent).key === "Backspace" &&
+                  reason === "removeOption"
+                ) {
+                  return
+                }
+                setAnchorEl(null)
+                if (newValue?.symbol) {
+                  onSelect?.(newValue?.symbol)
+                }
+              }}
+              renderOption={(props, option) => (
+                <ListItem listItemProps={props} {...option} />
+              )}
+              getOptionLabel={(option) => option.symbol}
+              getOptionDisabled={(option) => !option.isAvailable}
+              PopperComponent={(props) => (
+                <div {...props} data-testid="dropdownContainer"></div>
+              )}
+              PaperComponent={(props) => <Box {...props} marginRight={-2} />}
+              noOptionsText={"No tokens found."}
+              disabledItemsFocusable
+            />
+          </Box>
+        </ClickAwayListener>
+      </StyledPopper>
+    </div>
+  )
+}
+
+function ListItem({
+  amount,
+  valueUSD,
+  name,
+  icon,
+  symbol,
+  decimals,
+  isAvailable,
+  swapType,
+  listItemProps,
+}: TokenOption & {
+  listItemProps: React.HTMLAttributes<HTMLLIElement>
+}) {
+  const theme = useTheme()
+  const { t } = useTranslation()
+  const isVirtualSwap = (
+    [
+      SWAP_TYPES.SYNTH_TO_SYNTH,
+      SWAP_TYPES.SYNTH_TO_TOKEN,
+      SWAP_TYPES.TOKEN_TO_SYNTH,
+      SWAP_TYPES.TOKEN_TO_TOKEN,
+    ] as Array<SWAP_TYPES | null>
+  ).includes(swapType)
+  return (
+    <li
+      {...listItemProps}
+      data-testid="swapTokenItem"
+      style={{
+        paddingLeft: 0,
+        borderBottom: `1px solid  ${theme.palette.other.border}`,
```

Comment:
There are 2 spaces here between `solid` and `${theme`. Not sure if it breaks anything but better to make single for sake of consistency.